sdk/storage/Azure.Storage.Blobs/src/Models/UploadContent.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using Metadata = System.Collections.Generic.IDictionary<string, string>;
+
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Blob content to be uploaded by a <see cref="Specialized.BlobBaseClient"/>.
+    /// </summary>
+    public class UploadContent
+    {
+        /// <summary>
+        /// Content stream to upload.
+        /// </summary>
+        public Stream ContentStream { get; set; }
```

Comment:
We should call this just `Content` to be consistent with other `Stream content` parameters, `BlobDownloadInfo.Content`, etc.